superset-frontend/src/views/components/Menu.tsx

```diff
@@ -237,7 +237,7 @@ export function Menu({
     }
     return (
       <SubMenu
-        key={index}
+        key={`${index} - ${label}`}
```

Comment:
if these are guaranteed to be unique, let's just use the label here?

Reply:
Yes good idea, I changed it to `label`, Tnx